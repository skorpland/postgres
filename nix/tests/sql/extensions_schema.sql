-- all default extensions are installed in a schema "extensions"
-- we don't include the version as that will break often, we only care about
-- ensuring these extensions are present
select
    e.extname as extension_name,
    n.nspname as schema_name,
    e.extowner::regrole as extension_owner
from
    pg_extension e
join
    pg_namespace n on e.extnamespace = n.oid
where
    n.nspname = 'extensions' and e.extname != 'pgjwt'
order by
    e.extname;
