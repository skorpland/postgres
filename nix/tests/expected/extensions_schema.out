-- all default extensions are installed in a schema "extensions"
-- we don't include the version as that will break often, we only care about
-- ensuring these extensions are present
select
    e.extname as extension_name,
    n.nspname as schema_name,
    e.extowner::regrole as extension_owner
from
    pg_extension e
join
    pg_namespace n on e.extnamespace = n.oid
where
    n.nspname = 'extensions' and e.extname != 'pgjwt'
order by
    e.extname;
   extension_name   | schema_name | extension_owner 
--------------------+-------------+-----------------
 pg_stat_statements | extensions  | powerbase_admin
 pgcrypto           | extensions  | powerbase_admin
 uuid-ossp          | extensions  | powerbase_admin
(3 rows)

